src/ca_biositing/pipeline/tests/test_resource_transform.py

```
@@ -6,7 +6,7 @@
 # Mocking the database engine and session for normalize_dataframes
 @patch("ca_biositing.pipeline.utils.engine.engine")
 @patch("ca_biositing.pipeline.utils.name_id_swap.Session")
-def test_resource_transform(mock_session, mock_engine):
+def test_resource_transform(mock_engine, mock_session):
```

Ruff (0.15.1)

[warning] Unused function argument: `mock_engine`

(ARG001)